src/imagery/i.eodag/i.eodag.html

```diff
@@ -2,38 +2,272 @@ <h2>DESCRIPTION</h2>
 <h3>WARNING: I.EODAG IS UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</h3>
 
 <em>i.eodag</em> allows to search and download imagery scenes, e.g. Sentinel,
-Landsat, and MODIS, from a number of different providers.
-The module utilizes the
+Landsat, and MODIS, as well as other Earth Observation products, from a number 
+of different providers. The module utilizes the
 <a href="https://eodag.readthedocs.io/en/stable/">EODAG API</a>,
-as a single interface to search for scenes on the providers that EODAG supports.
-
+as a single interface to search for products within the supported providers.
 
 <p>
-Currently, only products which footprint intersects the current computational
-region extent (area of interest, AOI) are retrieved.
+By default <em>i.eodag</em> will search for products which footprint intersects
+the current computational region extent. Users can alternatively opt to pass a 
+vector map throught the <b>map</b> option to define the area of interest (AOI) 
+and change the relation with product footprints by means of the 
+<b>area_relation</b> or <b>minimum_overlap</b> options.
 
 <p>
 To only list available scenes, <b>l</b> flag must be set. If no <b>start</b>
 or <b>end</b> dates are provided, the module will search scenes from the
 past 60 days.
 
 <p>
-To download all scenes found within the time frame provided, the user must
+To download all scenes found within the time frame provided, users must
 remove the <b>l</b> flag and provide an <b>output</b> directory. Otherwise,
-files will be downloaded into <em>/tmp</em> directory.
+files will be downloaded into the /tmp directory.
 To download only selected scenes, one or more IDs must be provided
 through the <b>id</b> option.
 
 <p>
-To be able to download data through <em>i.eodag</em>, the user will need to register for the
-providers of interest.
-<em>i.eodag</em> reads the user credentials, which the user should have acquired,
-from the EODAG YAML config file.  User have to specify the config file path
-location through the <b>config</b> option, otherwise <em>i.eodag</em>
-will use the credentials found in the default config file <em>~/.config/eodag/eodag.yml</em>
+To be able to download data through <em>i.eodag</em>, users will
+need to register for the providers of interest.
+<em>i.eodag</em> reads user credentials from the EODAG YAML config file.  
+Users have to specify the config file path through the <b>config</b> 
+option, otherwise <em>i.eodag</em> will use the credentials found in 
+the default config file <em>~/.config/eodag/eodag.yml</em>
```

Comment:
```suggestion
the default config file <em>~/.config/eodag/eodag.yml</em> which is auto-generated the
first time eodag is used after an install in a local directory (`$HOME/.config/eodag/eodag.yml`
on Linux and BSD; XXXX on Windows; YYYY on Mac).
```

I suggest to add a note where the config is... (dunno where it would be on Windows or Mac).

Yet it looks like a chicken-egg problem: `pip install eodag` will install the software but what will generate the config? At first run on `i.eodag` if we do not want to suggest to run the `eodag` CLI tool? Perhaps I overlook something.

Reply:
The installation of eodag creates the file in the default location, then the user can modify it there or create a different one and place it somewhere else.

Reply:
> I suggest to add a note where the config is... (dunno where it would be on Windows or Mac).

I tested it on Mac, and it would be in the same location  `$HOME/.config/eodag`. Don't know about Windows though, yet.

> Yet it looks like a chicken-egg problem: `pip install eodag` will install the software but what will generate the config? At first run on `i.eodag` if we do not want to suggest to run the `eodag` CLI tool? Perhaps I overlook something.

Maybe we could add a flag `i`, for initialization, to use when running i.eodag for the first time. Just for the sake of making it less confusing, and then maybe we could use that flag to implement a custom GRASS GIS EODAG config file, if we need to.

Reply:
Can't that just happen within i.eodag (with a check or so) without another flag? Users should be able to create their config files or modify the default one. 

At this point, we need to move forward with the other objectives of the GSoC project.